src/emulator/storage/persistence.ts

```diff
@@ -54,6 +57,7 @@ export class Persistence {
   deleteFile(fileName: string, failSilently = false): void {
     try {
       unlinkSync(this.getDiskPath(fileName));
+      this._diskPathMap.delete(fileName);
```

Comment:
tiny nit: please only keep lines that can actually throw inside `try` blocks so it's clear what errors are being caught